test-requirements.txt

```diff
@@ -2,3 +2,5 @@ coverage
 glueviz
 matplotlib
 pytest
+vcrpy
+pathlib
```

Comment:
Why do we need pathlib?

Reply:
pathlib was added after updating the unit test to be consistent with NSLS-II/pyolog2/test_olog.py
 and in order to support the Python 2.7 test environment